agent-inject/agent/annotations.go

```diff
@@ -53,6 +53,10 @@ const (
 	// originated from.
 	AnnotationAgentRequestNamespace = "vault.hashicorp.com/agent-request-namespace"
 
+	// AnnotationAgentInitFirst makes the initialization container the first container
+	// to run when a pod starts.  Default is last.
```

Comment:
Extra space between two sentences should be removed.

Reply:
Done!